etna/ciim/utils.py

```diff
@@ -1,7 +1,8 @@
 import re
 
-from typing import Dict
+from typing import Any, Dict
 
+from django.template import TemplateSyntaxError, Variable, VariableDoesNotExist
```

Comment:
1. This is not standard in Django tough, is this taken from here https://github.com/django/django/blob/main/django/template/base.py

Reply:
I'm not sure what you mean @TNA-Allan? I'm importing from:
https://github.com/django/django/blob/main/django/template/__init__.py

Reply:
What I mean by its not standard that its not documented in Django Documentation to use `TemplateSyntaxError, Variable, VariableDoesNotExist`. I think it usage is on special needs only to extend a functionality. 

I notice a difference in accessing the api response path as a result of using `django.template`.

1. Is there anything which did not function as required in order to use this functionality of django.template?
2. Can this bit be substituted with anything else,  so that we not want any impact on our code if ever there are changes to the internal working of the template?